pom.xml

```diff
@@ -2,8 +2,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>21</maven.compiler.source>
```

Comment:
Please update the contributing readme as well https://github.com/PGMDev/PGM/blob/dev/docs/CONTRIBUTING.md?plain=1#L30